README.md

```diff
@@ -1001,13 +1002,93 @@ generate xccdf2inspec_stub              Translate a DISA STIG XCCDF XML file int
     -s, --singleFile                Output the resulting controls as a single file
 ```
 
+---
 
+### Supplement
 
+Supplement (ex. read or modify) elements that provide contextual information in the Heimdall Data Format results JSON file such as `passthrough` or `target`
```

Comment:
Nitpick here, but I think this wording is more clear

`Supplement (ex. read or modify) elements` ->
`Use the Supplement command to read or modify elements...`

Reply:
We've been trying to start with the verb every time.  If you have a suggestion that lets it stay that way, please let me know.